docs/configuration/modules/index.md

```diff
@@ -81,3 +81,4 @@ module2:
 - [security-analysis](security-analysis.md)
 - [simulation-parameters](simulation-parameters.md)
 - [table-formatter](table-formatter.md)
+- [network](network.md)
```

Comment:
These links are alphabetically ordered, could you please do the same? It makes searching for specific modules easier and allows us to check quickly if the page is updated when a new module is added.

Reply:
done